README.md

```diff
@@ -119,7 +119,11 @@ There are a lot of tests, so it can be helpful to constrain the tests you run lo
 
 ## Design Assets
 
-The CMS Design System provides a Sketch file and Sketch Library containing components, styles, and symbols. These are regularly updated alongside our code, and updates are automatically synced for designers using the Sketch Library.
+As of {MONTH}, 2024 the CMS Design System is using Figma as its primary design software. Designs existing in Sketch will continue to exist for a period of time. No new designs will be represented in Sketch, and links to Sketch design files will remain on the Doc Site until all teams are using Figma.
```

Comment:
I haven't checked the Jira ticket to see if this is the approved copy yet, but I wonder if the "As of MONTH, 2024" part is needed? Mostly because Git timestamps your changes and this seems like information that's not super useful from a README perspective.

Reply:
Yeah, I added this bit because I thought 'hey context might be nice'. But also if it doesn't fit the tone or expectations of a README I'm happy to cut it. 

Also did not realize the text content had been fully approved. Good to know! Thanks @zarahzachz

Reply:
Oh, I don't know if there was approved copy. I was just saying I hadn't read the ticket to see if there was approved copy. What you have seems fine.

One thing I may suggest is having your designers do a once over to make sure they didn't want to add something to this section.